ui/src/turing/components/configuration/standard_ensembler_config/RouteNamePathConfigGroup.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import {
+  EuiDescriptionList,
+  EuiDescriptionListDescription,
+  EuiDescriptionListTitle,
+  EuiTextColor,
+  EuiTitle,
+} from "@elastic/eui";
+
+export const RouteNamePathConfigGroup = ({ routeNamePath }) => {
+  return (
+    <EuiTitle size="xs">
+      <EuiTextColor>
+        <EuiDescriptionList
+          textStyle="reverse"
+          type="responsiveColumn"
+          compressed>
```

Comment:
Shall we use a map format for this instead? 
```
const routeNamePathConfig = [
  {
    title: 'Route Name Path',
    description: routeNamePath,
  }
]
```

With the `textStyle="reverse"` setting, I think the title shouldn't have been bold? Hence, was wondering if the above format would help achieve this instead.
![image](https://user-images.githubusercontent.com/25025366/191670776-50033dee-a447-4cf3-af2d-80cdcc3d2c8d.png)

Reply:
Hmm I think I was going for the same look as this total number of experiments look but I guess we can make it look like the fallback panel too:

![Screenshot 2022-09-22 at 6 46 03 PM](https://user-images.githubusercontent.com/36802364/191727214-df19d828-d481-4c38-b993-2dd1a8d6dce1.png)

Reply:
And no apparently it seems like changing to the map format does not work either. It's probably got to do with some lower-level styling because the panel showing the total number of experiments is like that too:

![Screenshot 2022-09-22 at 6 52 03 PM](https://user-images.githubusercontent.com/36802364/191728332-6c63e991-3b71-4ff1-a52e-b080fb37022b.png)